src/javascript/app_2/App/Components/Form/DatePicker/date_picker.jsx

```diff
@@ -197,18 +230,32 @@ class DatePicker extends React.Component {
                     })}
                     onClick={this.state.is_clear_btn_visible ? this.clearDatePickerInput : undefined}
                 />
-                <div
-                    className={classNames('datepicker__picker', {
-                        'datepicker__picker--show'                           : this.state.is_datepicker_visible,
-                        [`datepicker__picker--align-${this.props.alignment}`]: this.props.alignment,
-                    })}
+                <CSSTransition
+                    in={this.state.is_datepicker_visible}
+                    timeout={100}
+                    classNames={{
+                        enter    : 'datepicker__picker--enter',
+                        enterDone: 'datepicker__picker--enter--done',
```

Comment:
Maybe `enterDone` modifier class should called `--enter-done` instead? It's actually a separate modifier `class` from `--enter`.

Reply:
ok, sure